assets/dev/js/editor/components/icons-manager/classes/icon-library.js

```diff
@@ -0,0 +1,152 @@
+import Store from './store';
+
+const IconLibrary = class {
```

Comment:
`export default class`

Usually you should export the class and make a new instance of it in the inclusive file

Reply:
Moved to elementor.iconManager.library